README.md

```diff
@@ -3,21 +3,91 @@
 # Welcome to Sign
 
 ## Brief intro to the repo 
-This is a Client Server Application to upload images to server/ S3 bucket
+This is an admin dashboard application for a school attendance marking system. Essentially, a Client Server Application to upload images to server/ S3 bucket. 
```

Comment:
Lets remove the "a Client Server Application to upload images to server/ S3 bucket." part